README.md

```diff
@@ -26,6 +26,10 @@ This skill has some spicy language options. It starts in grandma mode by default
 
 If you want to go back into grandma mode, say "I'm too young to die."
 
+## Requirements
+
+Most sounds will play on any device, but longer sounds require OCP to be enabled.
```

Comment:
i think this will be more confusing than helpful for new users, they will think some extra step is needed for OCP

Reply:
perhaps a comment that music intents might not work properly in hivemind?